grakn-graph/src/main/java/ai/grakn/graph/internal/ValidateGlobalRules.java

```diff
@@ -275,6 +279,35 @@ private ValidateGlobalRules() {
         return Optional.empty();
     }
 
+    static Set<String> validateRuleIsHornClause(GraknGraph graph, Rule rule){
```

Comment:
Javadoc what this rule actually checks. I.e. state what a horn clause it and how you check the rule is one. Validtor gets confusing fast without very clear javadocs.